fft/src/helpers.rs

```diff
@@ -1,10 +1,16 @@
-use super::errors::FFTError;
+use lambdaworks_math::field::{element::FieldElement, traits::IsField};
 
-pub fn log2(n: usize) -> Result<u64, FFTError> {
-    if !n.is_power_of_two() {
-        return Err(FFTError::InvalidOrder(
-            "The order of polynomial + 1 should a be power of 2".to_string(),
-        ));
+pub fn next_power_of_two(n: usize) -> usize {
```

Comment:
please add two unit tests, one for an input that is a power of two and one for a number that isn't.

Reply:
Done!